### ISSUE TITLE
Experimental setup for comparison with your work

### ISSUE THREAD

**Issue by wangzhen38** [ruizhang-ai/L0-SIGN_Detecting_Benefitial_Feature_Interactions_for_Recommender_Systems#3]
Dear researchers,
I read your paper and it is so interesting. However, I would try to reproduce your experiments and I have on question: I run your code but just achieve the results(auc:0.8942, acc:0.8855 on movie-tags) with 300 epochs,  i want to know how to achieve the experiment data in your paper.

Thank you very much,
wangzhen38

**Comment by ethanmock** [comment]
Hi wangzhen38,

Thanks for being interested in our work. 

The results look strange. Sometimes the random shuffle (the training set and test set are different at each run) of the dataset may introduce selective bias, but the main reason may be the hyper-parameter setting.
L0-SIGN has a problem of being unstable when training with the L0 regularization. Sometimes the training process may collapse or converge to a suboptimal result due to an unsuitable hyper-parameter setting (e.g., unsuitable L0 weight) or even bad luck of parameter initialization (e.g., the remaining edges are too few at the beginning). We are still focusing on stabilizing the training process. However, you can adjust the hyper-parameters to get expected results. For example, change the L0 and L2 weights (usually search from 1e-5 to 1e-1 and adapt the corresponding learning rate).
